pkg/admissioncontroller/webhook/admission/updaterestriction/add.go

```diff
@@ -0,0 +1,29 @@
+// SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Gardener contributors
```

Comment:
As the PR is opened in 2025

```suggestion
// SPDX-FileCopyrightText: 2025 SAP SE or an SAP affiliate company and Gardener contributors
```